modules/standard/List.chpl

```diff
@@ -151,7 +151,7 @@ module List {
     type eltType;
 
     /*If `true`, this list will perform parallel safe operations.*/
-    @unstable("'list.parSafe' is unstable and is expected to be replaced by a separate list type in the future");
+    @unstable("'list.parSafe' is unstable and is expected to be replaced by a separate list type in the future")
     param parSafe = false;
```

Comment:
Ditto here about `;` on `parSafe`.

Reply:
This one I know we have tests of, the `;` had no affect other than being stylistically wrong

I suppose we could argue that attributes ignoring `;` is a bug, but I am not inclined to make that argument

Reply:
It's a pleasant-surprise sort of bug -- or maybe a feature someone deliberately added? Anyway, nothing for you to do.